components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -349,7 +349,8 @@ class TabsUseCases(
         ) = withContext(Dispatchers.IO) {
             val now = System.currentTimeMillis()
             val state = storage.restore {
-                now - it.lastAccess <= tabTimeoutInMs
+                val lastActiveTime = maxOf(it.lastAccess, it.createdAt)
+                now - lastActiveTime <= tabTimeoutInMs
```

Comment:
I always wondered if a new tab should start with `lastAccess = current time` - but this here seems to be a simpler workaround?

Reply:
That's a really good point. I would like to remove two concepts and just use one. 🤔 

Looking at some designs, I can see Fenix's tabs tray requiring `createdAt` when we start displaying the tray by created time.

Reply:
I'm starting to like this idea a lot more as I think about it. It will solve a few problems for me in the tabs tray.

_needs to see if it will cause regressions_

Reply:
While looking at another bug, Christian suggested we stick with the maxOf direction instead of changing the default for two reason:
1. We have existing features (Recently closed) that depend on the current behaviour of 0 being the default.
2. This aligns with what iOS also has.

Reply:
1. Okay, makes sense for now.
2. I don't think that is a strong reason. :)

Reply:
Sorry, I didn't explain 2 properly: there are features we will want to release on Android/iOS which are based on the concepts of lastUsed / createdAt , and while we aren't strictly tied to following them, it would be easier if we kept them similar. That's as far as I understand it at least. 🙂